src/utils/env.ts

```diff
@@ -4,11 +4,14 @@
 import execa from 'execa';
 import stripAnsi from 'strip-ansi';
 
+import additionalEnvs from './env.json';
+
 const args = ['-ilc', 'echo -n "_SHELL_ENV_DELIMITER_"; env; echo -n "_SHELL_ENV_DELIMITER_"; exit'];
 
 const ENV = {
   // Disables Oh My Zsh auto-update thing that can block the process.
   DISABLE_AUTO_UPDATE: 'true',
+  ...additionalEnvs,
```

Comment:
do we need to add these to `getSerializedProcessEnv`  in electron/utils.ts - so they are passed in the IPC call that sets the main process env in the renderer thread?

Reply:
We currently don't use these environment values in the renderer thread. I think it is better to add when we need it

Reply:
ok - but we shouldn't be calling shellEnvSync() in the main thread - this method was created for the renderer thread!?

Reply:
seems right!